src/Native/Utils.js

```diff
@@ -357,6 +357,10 @@ Elm.Native.Utils.make = function(localRuntime) {
 
 	var toString = function(v)
 	{
+		if (v === null)
+			{
+				return '<internal structure>';
+			}
```

Comment:
You are right, there are only tabs in the code. However, you still have unnecessary indent for `if` body. Brace `{` should be below `i`.